PaddleCV/image_classification/dist_train/run_nccl2_mode.sh

```diff
@@ -1,4 +1,26 @@
 #!/bin/bash
+set -e
+
+enable_dgc=False
```

Comment:
Does this mean only nccl distributed mode supports DGC?

Reply:
Ye